octobot_trading/constants.py

```diff
@@ -42,6 +42,9 @@
 DEFAULT_BACKTESTING_TIME_LAG = 0
 INFINITE_MAX_HANDLED_PAIRS_WITH_TIMEFRAME = -1
 
+FULL_HISTORY_EXCHANGES = ["bequant", "binance", "binanceus", "binanceusdm", "bitcoincom",
```

Comment:
@valouvaliavlo You mean full candle history for backtesting right ?

Reply:
yes, it's that

Reply:
Ok :+1:
Can you rename it with : 
```suggestion
FULL_CANDLE_HISTORY_EXCHANGES = ["bequant", "binance", "binanceus", "binanceusdm", "bitcoincom",
```
please ?